build/ApiDiff.props

```diff
@@ -3,6 +3,7 @@
     <ApiContractPackageVersion>0.10.0</ApiContractPackageVersion>
     <NugetPackageName Condition="'$(PackageId)' != ''">$(PackageId)</NugetPackageName>
     <NugetPackageName Condition="'$(PackageId)' == ''">Avalonia</NugetPackageName>
+	<RunApiCompat Condition="'$(TargetFramework)' == 'net6.0'">false</RunApiCompat>
```

Comment:
Don't run ApiCompat on `net6` as we have no packages to compare against.

Reply:
In general, it is better to use the following for this :)

`$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))`

Reply:
Maybe it’s better to use the following for this?

`$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))`

> Return 'True' if the candidate target framework is compatible with this target framework and false otherwise.